assistant/v2.ts

```diff
@@ -519,8 +519,20 @@ namespace AssistantV2 {
   export interface MessageContextSkill {
     /** Arbitrary variables that can be read and written by a particular skill. */
     user_defined?: JsonObject;
-    /** For internal use only. */
-    system?: JsonObject;
+    /** System context data used by the skill. */
+    system?: MessageContextSkillSystem;
```

Comment:
Does generating with the new API def tag still generate this change?

Reply:
Re-generated the SDK, this change should not be present anymore